python/ray/experimental/dag/class_node.py

```diff
@@ -87,6 +87,10 @@ def __getattr__(self, method_name: str):
     def __str__(self) -> str:
         return get_dag_node_str(self, str(self._body))
 
+    def bind(self, *args, **kwargs):
```

Comment:
cc: @ericl

Reply:
This is inconsistent with the Ray core API